Sources/protoc-gen-swiftgrpc/Generator.swift

```diff
@@ -106,35 +99,14 @@ class Generator {
     println()
     
     if options.generateClient {
-      if options.generateImplementations {
-        for service in file.services {
-          self.service = service
-          printClient(asynchronousCode: options.generateAsynchronous,
-                      synchronousCode: options.generateSynchronous)
-        }
-      }
-
-      if options.generateTestStubs {
-        if options.generateNIOImplementation {
-          fatalError("Generating test stubs is not yet supported for SwiftGRPC-NIO.")
```

Comment:
I think this is still true. (At least we will probably need to generate client test stubs at some point.)

Reply:
It is true, but I didn't find any NIO test stubs so I thought it was better to remove the option than have an option which didn't do anything!